src/main/java/com/aws/greengrass/deployment/DeploymentDocumentDownloader.java

```diff
@@ -167,6 +177,14 @@ private GetDeploymentConfigurationResponse getDeploymentConfiguration(String dep
 
             deploymentConfiguration = greengrassServiceClientFactory.fetchGreengrassV2DataClient()
                     .getDeploymentConfiguration(getDeploymentConfigurationRequest);
+        } catch (AccessDeniedException e) {
+            throw new DeploymentTaskFailureException("getDeploymentConfiguration API returned 403 access denied. "
```

Comment:
nit: GetDeploymentConfiguration to match the IoT policy grant statement. Also similar message as before about format.

Reply:
will update